### ISSUE TITLE
preprocessor removes characters 'xp' from English words

### ISSUE THREAD

**Issue by olihawkins** [s/preprocessor#14]
```python
import preprocessor as p

# Returns 'elosion'
p.clean('explosion')

# Returns 'eerience'
p.clean('experience')
```

**Comment by kLabille** [comment]
I've got the same problem.

It does not happen if you manually remove the p.OPT.SMILEY from the option: 
p.set_options(p.OPT.URL, p.OPT.EMOJI, p.OPT.MENTION, p.OPT.HASHTAG, p.OPT.NUMBER)

But in that case every smileys will remain in the tweet...

**Comment by olihawkins** [comment]
Have found it also happens with 'xo' ...

```python
# Returns 'enerate'
p.clean('exonerate')
```

**Comment by kLabille** [comment]
I fixed it buy adding a space at the beginning of the smileys regex in the defines.py file
It works for what I need to do

**Comment by olihawkins** [comment]
That's helpful. Thanks! Will give it a go.

**Comment by kLabille** [comment]
You're welcome. Let me know if you have trouble doing it.

**Comment by SefaraTJ** [comment]
Possible fix in #23. Close #14

**Comment by s** [comment]
Fixed in `0.6.0`
